packages/prisma/schema.prisma

```diff
@@ -608,6 +608,7 @@ model CreditExpenseLog {
   date            DateTime
   smsSid          String?
   smsSegments     Int?
+  callDuration    Int?
```

Comment:
Might be better to introduce a _creditFor_ or similar field that defines what the credit is for.

It seems like existing entries are for SMS and _callDuration_ is for _CALL_.

In this way we could ensure in the Repository to have a required _callDuration_ value, providing better type safety or assertions

Reply:
@calcom/dba for review and suggestions